api-report/sequence.api.md

```diff
@@ -382,6 +403,20 @@ export interface IValueOpEmitter {
     emit(opName: string, previousValue: any, params: any, localOpMetadata: IMapMessageLocalMetadata): void;
 }
 
+// @public (undocumented)
+export interface PositionSpec {
+    // (undocumented)
+    end?: number;
```

Comment:
these shouldn't be nullable. It'd probably be good to add a compile-time test case somewhere that `.change` doesn't accept specifying a single property out of start/end with [ts-expect-error](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) (just put it in a block of code that doesn't execute, e.g. a function that's never called)